### PR TITLE
Improvement: Add Ruff configuration

### PR DESCRIPTION
**Pull request by snova-francescar**
Cf. this [confluence page](https://sambanova.atlassian.net/wiki/spaces/AIS/pages/1590820987/Python+best+practices) for reference.

### CODE REVIEW

File: settings.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (snova-francescar) [inline settings.json:1]: @snova-kwasia, @snova-jorgep 
We probably should not commit the VSCode `settings.json`.
Everyone should be free to configure it as they wish.

Comment (snova-francescar) [reply settings.json:1]: We can move this to Confluence.

Comment (snova-jorgep) [reply settings.json:1]: I agree with this, is better to have among the explanations you have on the confluence page

**Review by snova-jorgep — APPROVED** [review]

File: settings.json (hunk 1 above)
Comment (snova-kwasia) [reply settings.json:1]: Yep agreed that confluence is the best place for this.

**Review by snova-kwasia — APPROVED** [review]
Looks good @snova-francescar , cheers and approved.

**Comment by snova-kwasia** [comment]
Merged.

**Pull request merged**